src/webrtc/connection.ts

```diff
@@ -18,8 +18,8 @@ import type { Stream, StreamResult, DialOptions } from '../types'
 const DEBUG_PREFIX = `hopr-connect`
 
 const _log = Debug(DEBUG_PREFIX)
-const _verbose = Debug('hopr-connect:verbose')
-const _error = Debug(DEBUG_PREFIX.concat(`error`))
+const _verbose = Debug(DEBUG_PREFIX.concat(':verbose'))
```

Comment:
out of curiosity, wouldn't `\`${DEBUG_PREFIX}:verbose\`` work as well?

Reply:
think so, also easier to read. changing it